### ISSUE TITLE
Configure RDS Backups

### ISSUE THREAD

**Issue by UncleGedd** [defenseunicorns/uds-package-dubbd#335]
As a delivery engineer
If the RDS database ever goes down, becomes corrupted, etc
I want the capability to restore from a backup

### Notes
- Ensure we are taking RDS backups daily (frequency is debatable)

**Comment by justin-o12** [comment]
Is RDS currently or will be used anywhere?  I understand IDAM at some point eventually.

**Comment by justin-o12** [comment]
RDS will backup daily during the backup window as long as the retention period is >= 1 day.
[UDS module source location](https://github.com/defenseunicorns/terraform-aws-uds-rds/blob/main/main.tf#L33-L38)
[AWS docs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithAutomatedBackups.html#USER_WorkingWithAutomatedBackups.Enabling)

[It's also possible to replicate into another AWS region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ReplicateBackups.html), but more work needs to be done to enable this in our UDS module.

**Comment by justin-o12** [comment]
For restoring:
>You can't restore from a DB snapshot to an existing DB instance; a new DB instance is created when you restore. 

>You can restore a DB instance to a specific point in time, creating a new DB instance.
But,
>You can restore to any point in time within your backup retention period.

**Comment by justin-o12** [comment]
TL;DR the current state of the UDS RDS module has backups enabled in the same region in RDS using a 1 day retention period, @UncleGedd

**Comment by UncleGedd** [comment]
👍 perfect!
